### PR TITLE
os/bluestore: make bdev label parsing error more meaningful and less noisy

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -4200,10 +4200,10 @@ int BlueStore::_read_bdev_label(CephContext* cct, string path,
     decode(expected_crc, p);
   }
   catch (buffer::error& e) {
-    derr << __func__ << " unable to decode label at offset " << p.get_off()
+    dout(2) << __func__ << " unable to decode label at offset " << p.get_off()
```
Comment (varadakari) [inline src/os/bluestore/BlueStore.cc]: dout(1), still an error, not able to decode.

**Comment by alfredodeza** [comment]
There isn't a way to conclude that this is the first time the OSD is being created which means it will hit this error? Ideally it wouldn't just send anything out. My concern is that if we make it less noisy, we won't catch it when it is a real problem

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-01-31_01:48:16-rados-wip-kefu-testing-2018-01-31-0034-distro-basic-mira/

failure tracked at http://tracker.ceph.com/issues/22846

@alfredodeza up to you to hold it back or merge it.

**Comment by alfredodeza** [comment]
Actual ticket is http://tracker.ceph.com/issues/22285

If there isn't a way to make this understand this is the first time starting up, then I guess we can just move forward with this. Anything is better than what we currently have that looks like an error :(

**Comment by liewegas** [comment]
I think this is good as is!

**Comment by alfredodeza** [comment]
How can we get this backported to Luminous?

**Comment by alfredodeza** [comment]
Never mind, backported to Luminous with https://github.com/ceph/ceph/pull/20326

**Pull request merged**